runtime/runtime/src/verifier.rs

```diff
@@ -153,8 +153,8 @@ pub fn verify_and_charge_transaction(
         }
     };
 
-    set_access_key(state_update, &signer_id, &transaction.public_key, &access_key);
-    set_account(state_update, &signer_id, &signer);
+    set_access_key(state_update, signer_id.clone(), transaction.public_key.clone(), &access_key);
```

Comment:
I guess this is because copying is usually advised to be bubbled up in the call hierarchy.